src/components/ContentSection.tsx

```diff
@@ -1,24 +1,31 @@
 import { HTMLAttributes, PropsWithChildren } from 'react'
 import { twMerge } from 'tailwind-merge'
 
-export const ContentSection = ({
+interface ContentSectionProps
+  extends PropsWithChildren,
+    Omit<HTMLAttributes<HTMLDivElement>, 'className'> {
+  backgroundVariant?: 'gray'
+  removePadding?: boolean
```

Comment:
I'd argue an option that enables padding (which would be true by default) would be easier to reason with. `padding` or `withPadding` I guess. Feels a bit weird to _enable_ an option that _disables_ something.